package.json

```diff
@@ -14,6 +14,19 @@
   "main": "lib/cjs/index.js",
   "types": "lib/cjs/index.d.ts",
   "module": "lib/esm/index.js",
+  "type": "commonjs",
+  "exports": {
+    ".": {
+      "types": "./lib/cjs/index.d.ts",
+      "import": "./lib/esm/index.js",
+      "require": "./lib/cjs/index.js"
+    },
+    "./*": {
+      "types": "./lib/cjs/*.d.ts",
+      "import": "./lib/esm/*.js",
+      "require": "./lib/cjs/*.js"
+    }
```

Comment:
This solves deep imports, so `cherry-pick` is not necessary.

Reply:
We can't remove cherry pick because we need to support versions of webpack, node, etc that don't support exports in pkg.json at least not without a major bump

Reply:
I can bring it back.

Reply:
Yeah, we need some way of testing it tho. Maybe a node smoke test?

Reply:
> We can't remove cherry pick because we need to support versions of webpack, node, etc that don't support exports in pkg.json at least not without a major bump

I've brought back the `cherry-pick` build with 048959a4f494a25aca3205f2662f6b04d858205c

Reply:
> Yeah, we need some way of testing it tho. Maybe a node smoke test?

There was a basic smoke test (`test:build`). I've improved it further to make sure that the built files are actually ESM and CommonJS modules. Please see 69a6dc62ca7d02ef4ede046071f1262c62c95bb1